syntax = "proto3";

// option java_multiple_files = true;
// option java_package = "io.grpc.examples.helloworld";
// option java_outer_classname = "DFSProto";
// option objc_class_prefix = "HLW";

package dfs;

// The greeting service definition.
service DFS {
  // Sends a greeting
  rpc CreateFile (CreateFileRequest) returns (CreateFileReply) {}
}

// The request message containing the user's name.
message CreateFileRequest {
  string path = 1;
}

// The response message containing the greetings
message CreateFileReply {
  string message = 1;
}
